ci-operator/config/opendatahub-io/vllm/opendatahub-io-vllm-main.yaml

```diff
@@ -101,16 +63,7 @@ tests:
       SOURCE_IMAGE_REF: vllm-build-main-cuda
     env:
       IMAGE_REPO: vllm
-      RELEASE_VERSION: cuda-fast
-    workflow: opendatahub-io-ci-image-mirror
-- as: rocm-fast-image-mirror
-  postsubmit: true
-  steps:
-    dependencies:
-      SOURCE_IMAGE_REF: vllm-build-main-rocm
-    env:
-      IMAGE_REPO: vllm
-      RELEASE_VERSION: rocm-fast
+      RELEASE_VERSION: fast-cuda
```

Comment:
```suggestion
      RELEASE_VERSION: fast
```

Reply:
we had already discussed about it in below slack thread :
https://redhat-internal.slack.com/archives/C07A8ECBTPB/p1725891663670459